src/app/ngrx/actions/cms.action.creator.ts

```diff
@@ -21,3 +21,12 @@ export class CmsAccountFailureAction implements Action {
   readonly type = ActionTypes.CMS_ACCOUNT_FAILURE;
   constructor(public payload: any) {}
 }
+
+export interface CmsAccountRetryActionsPayload {
```

Comment:
Hmm... I'm confused what this is.  I thought all the logic to display the user login was in Styleguide now - and that it hits `id.prx.org/userinfo` instead of the auth/accounts endpoint in CMS.

Reply:
In fact, I do see the ajax firing to get id-userinfo.  But I also see the requests going to CMS.

Reply:
It does the userinfo request from the styleguide as well. That gets the username for the dropdown menu. But we also need the user image, which I am pretty sure I can only get from CMS. Honestly, I just left the CMS stuff alone when I pulled out the podcast and episode requests to use castle instead.

Reply:
Ah, okay, so remembering more now.  The Publish AppComponent was updated to the user info service instead of CMS.  And that shared user-info-service can actually fetch the userimage itself.  (ID provides [a link](https://id.prx.org/userinfo?scope=profile%20apps) directly to your CMS public user, which has an image in it).

I think Metrics just got missed during that auth cleanup round.

Here it is -> https://github.com/PRX/publish.prx.org/pull/507

Reply:
ah... can we make that a separate ticket?